packages/web3-constants/evm/coingecko.json

```diff
@@ -0,0 +1,30 @@
+{
+    "PLATFORM_ID": {
+        "Mainnet": "ethereum",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "",
+        "Gorli": "",
+        "BSC": "binance-smart-chain",
+        "BSCT": "",
+        "Matic": "polygon-pos",
+        "Mumbai": "",
+        "Arbitrum": "arbitrum-one",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "xdai"
+    },
+    "COIN_ID": {
+        "Mainnet": "ethereum",
+        "Ropsten": "",
+        "Rinkeby": "",
+        "Kovan": "",
+        "Gorli": "",
+        "BSC": "binancecoin",
+        "BSCT": "",
+        "Matic": "matic-network",
+        "Mumbai": "",
+        "Arbitrum": "",
+        "Arbitrum_Rinkeby": "",
+        "xDai": "xdai"
+    }
+}
```

Comment:
I had thought about refactor these kind of constants data in https://jsonnet.org/

Reply:
It seems that we don't need to add logic for JSON files here.

Reply:
The logic is compile-time only.
Anyway, the thought is not mature yet.